doc/make.jl

```diff
@@ -178,18 +178,76 @@ const PAGES = [
 ]
 end
 
+const use_revise = "revise=true" in ARGS
+if use_revise
+    let revise_env = joinpath(@__DIR__, "deps", "revise")
+        Pkg.activate(revise_env)
+        Pkg.add("Revise")
+        Base.ACTIVE_PROJECT[] = nothing
+        pushfirst!(LOAD_PATH, revise_env)
+    end
+end
+function maybe_revise(ex)
+    use_revise || return ex
+    STDLIB_DIR = Sys.STDLIB
+    STDLIBS = filter!(x -> isfile(joinpath(STDLIB_DIR, x, "src", "$(x).jl")), readdir(STDLIB_DIR))
+    return quote
+        $ex
+        using Revise
+        const STDLIBS = $STDLIBS
+        Revise.track(Core.Compiler)
+        Revise.track(Base)
+        for (id, mod) in Base.loaded_modules
+            if id.name in STDLIBS
+                Revise.track(mod)
+            end
+        end
```

Comment:
Does this re-run `track` on each doctest in the same module? I think `track` is re-entrant safe now, but on the face of it this seems potentially wasteful.

Reply:
Since this is passed to `DocMet.setdocmeta!`, I think it should only get run once for each stdlib.